### PR TITLE
doc: provide more detailed instructions

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
provide more detailed instructions.

### What is changed and how it works?
1. mention version limit
2. provide URL for MySQL settings

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variables. Variables and their default values are showed in following table.
+2. Setup two MySQL servers (<5.8.0) with binlog enabled first and set `GTID_MODE=ON`, export proper environment variables. Variables and their default values are showed in following table.
```
Comment (GMHDBJD) [inline tests/README.md:36]: Maybe <8.0 will be better? Actually we currently not support mysql 8.0 both test and production environment.

Comment (lance6716) [reply tests/README.md:36]: noticed current dependence of `tidb-tools@v4.0.2-0.20200622075412-c409811814de+incompatible/pkg/check/mysql_server.go`

```
// SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
// * 5.6.0 <= MySQL Version < 5.8.0
// * 10.1.2 <= Mariadb Version
var SupportedVersion = map[string]struct {
	Min MySQLVersion
	Max MySQLVersion
}{
	"mysql": {
		MySQLVersion{5, 6, 0},
		MySQLVersion{5, 8, 0},
	},
	"mariadb": {
		MySQLVersion{10, 1, 2},
		MaxVersion,
	},
}
```

This PR is mainly caused by long error log which is hard to locate when I run integration test 😢

**Comment by lance6716** [comment]
quite a short PR! come and take a look @GMHDBJD @WangXiangUSTC

File: tests/README.md (hunk 1 above)
Comment (WangXiangUSTC) [reply tests/README.md]: there is no MySQL's version named `5.8.0`, how about changing it to `<=5.7`?

Comment (lance6716) [reply tests/README.md]: > there is no MySQL's version named `5.8.0`, how about changing it to `<=5.7`?

OK, going to change it to `5.6~5.7`

**Comment by WangXiangUSTC** [comment]
LGTM

**Comment by lance6716** [comment]
need 1 approving @WangXiangUSTC @GMHDBJD

**Pull request merged**